tests/installation/isosize.pm

```diff
@@ -11,7 +11,12 @@ sub run {
     if (!$size || !$max || $size > $max) {
         $result = 'fail';
     }
-    bmwqemu::diag("check if actual iso size $size fits $max: $result");
+    if (!defined $size) {
```

Comment:
I suspect this will not give you the result you are looking for, $size is defined earlier anyways, either real size of $iso or 0, the defined check here just useless IMO.